source/docs/articles/sites/domains.md

```diff
@@ -92,6 +92,17 @@ The provided IP address resolves to our load-balancers. When a request comes in,
 ### A contrib module that I use for my Drupal site does not support IPv6; how should I proceed?
 [Use the issue queue](https://drupal.org/node/317) of the module in question to communicate with the module maintainers.
 
+### Does Pantheon do any html domain substituion?
+Yes, for WordPress sites (which are more likely to contain hard-coded linked to other domains in their database), Pantheon will replace platform-assigned domains for that environment (i.e. `dev-mysite.pantheon.io` or `live-mysite.gotpantheon.com`) with custom domains in the HTML body, when the HTML page is requested via the custom domain.  This prevents loading assets using different domain names representing the same site.  
+
+For example, when requesting `http://mysite.com` in a browser, we would replace any asset (images, CSS, JS) URL domains from `live-mysite.pantheon.io` to `mysite.com`.
+
+This will not:
+
+* Effect Drupal
+* Substitute https domains
+* Substitute platform domains from another environment (i.e. `dev-mysite.pantheon.io` will not be substituted for `mysite.com` if that is a custom domain for the `live` environment)
```

Comment:
Should be "e.g." instead. "i.e." means a restatement in other words; "e.g." indicates an example.